components/src/04-views/catalog/Catalog.stories.js

```diff
@@ -27,7 +26,24 @@ storiesOf('Views/Catalog', module)
         activeFilterItems,
         checkboxItems,
         toolbar,
-        breadcrumbs
+        breadcrumbs: [
```

Comment:
Let's move it to `components/mocks/breadcrumbs.json` and use globally (for now in catalog, productView and breadcrumbs stories)

Reply:
Right now the mock is in `Breadcrumbs` component folder, I think it'll be better to move it into global mocks in `components/mocks/`